src/onboarding/Onboarding/Onboarding.js

```diff
@@ -246,6 +292,15 @@ function Onboarding({ status, selectorNetworks, walletWeb3, web3 }) {
           onConnect={handleProviderConnect}
           onConnectError={connectProviderError}
         />
+        {requirementsError[0] === 'wrong-network' ? (
```

Comment:
Only render the Modal if there's an actual Error, otherwise it have nothing to work on